local-deployment/README.md

```diff
@@ -72,9 +89,25 @@ bash exit_validators.sh -i validator_index -m "memonic" -b "http://bn_api_host:b
 ```
 
 # Enabling withdrawals
-The script `add_withdrawal_address.sh` allows you to create & submit a bls to execution layer address change message in order to enable withdrawals for specific validators.. Requires [/github.com/protolambda/eth2-val-tools](eth2-val-tools) to be available on the path.
+First, install _JQ_ and _eth2-val-tools_:
 ```
+sudo apt install jq
 go get github.com/protolambda/eth2-val-tools@master
 go install github.com/protolambda/eth2-val-tools@master
-bash add_withdrawal_address.sh -a "EL address (0x1234...)" -i validator_index -m "memonic" -b "http://bn_api_host:bn_api_port"
 ```
+To enable withdrawals for specific validators in your local network, we provide script `add_withdrawal_address.sh`. It creates and submits a BLS-to-execution-layer-address-change message.
+The script needs some arguments: 
+```
+cd ~/eth2-beaconchain-explorer/local-deployment/scripts
+bash add_withdrawal_address.sh -a "EL wallet address (0x1234...)" -m "memonic" -b "http://localhost:bn_api_port" -i validator_index
+```
+To fill the address field, you will find address generators with Google but you can also peek any existing address, like your own to enjoy watching it receive huge rewards locally.
+To fill the mnemonic field, copy it from our file _network-params.json_.
+Replace `bn_api_port` with the port that Kurtosis maps the http port of the consensus client to. This port is written on the console by Kurtosis when it starts, on this line: `cl-1-lighthouse-geth http: ...../tcp -> http://127.0.0.1:XXXXX` (what you want is represented by XXXXX).
+Finally, `validator_index` identifies the local validator that you want to activate withdrawals for. Give a number between 1 and 64.
```

Comment:
**Minor:** I feel like this is too much prose and some info is missing/wrong.

```suggestion
- `-a`: Followed by a `string` representing an ETH address prefixed with "0x".
- `-m`: Followed by a `string` representing the validator mnemonic. The one used to create the initial set of validators can be found in the _network-params.json_ file, see `preregistered_validator_keys_mnemonic`.
- `-b`: Followed by a `string` representing the URL to the consensus client. The whole URL is logged by Kurtosis when it starts. Search for `cl-1-lighthouse-geth` and look for something like `tcp-discovery: 30303/tcp -> 127.0.0.1:32787`. 
- `-i`: Followed by a `number` representing the index of the validator you want to activate withdrawals for. The amount of validators created can be also found in the _network-params.json_ file, see `num_validator_keys_per_node`. For 64 validators, the indexes range from 0 to 63.
```

Reply:
-a : I would make clear to the beginner (again) who doesn't master Ethereum yet that he can use any address, for example any one that he will generate with an online tool.

-b : You tell to search for the section reporting the mapping of the consensus client but you tell to pick an URL handled by the execution client. Moreover, this URL is used by the discovery service of the P2P protocol, I am not sure that it can receive CL/EL messages. The only address which works on my computer is on the line that I indicated (`http: 4000/tcp -> http://127.0.0.1:32779`).

Reply:
Regarding -a: I would argue that an Ethereum beginner does not want to launch a local testnet with our explorer. This is advanced.

Regarding -b: Jeah, sorry, copied the wrong line. Good catch :+1: